lib/main_common.pm

```diff
@@ -444,7 +444,6 @@ sub load_extra_tests() {
             # start extra x11 tests from here
             loadtest 'x11/vnc_two_passwords';
             # TODO: check why this is not called on opensuse
```

Comment:
No, this is also related to loadtest 'x11/user_defined_snapshot';
Im working on it right now. Later I can remove the TODO-Comment.